src/hwcrypto.js

```diff
@@ -77,10 +75,9 @@ var hwcrypto = (function hwcrypto() {
             }
         }
         // TODO: remove
-    window.addEventListener('load', function(event) {
         // There's a timeout because chrome content script needs to be loaded
-        probe();
-    });
+    (window.addEventListener || window.attachEvent).call(window, 'load', probe, false);
```

Comment:
The third argument `useCapture` is optional in current browsers but used to be mandatory in pre-2012 browsers which might be still used.